src/common/mipmap_cache.c

```diff
@@ -932,11 +932,6 @@ void dt_mipmap_cache_get_with_caller(
   }
 }
 
-void dt_mipmap_cache_write_get_with_caller(dt_mipmap_cache_t *cache, dt_mipmap_buffer_t *buf, const uint32_t imgid, const int mip, const char *file, int line)
```

Comment:
This is part of an API (not a macro) I think we should keep it. Could well be needed and better keep the API than letting user's redefining it somewhere else.